network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -71,10 +71,12 @@
 
 {% block body_wrapped %}
 
+  <header>
   {% include "./primary_nav.html" %}
+  </header>
 
   <div class="underneath-screen-overlay">
-    <div class="category-nav">
+    <nav class="category-nav" title="category navigation">
```

Comment:
PNI nav includes not only links for categories but also `Home` and `About`. Can we remove this `title` and add `aria-label` to category related nav links so they can be more descriptive for assistive technology.

Reply:
we cannot remove the title: it is required to inform whatever accessibility tool is look at this what the purpose of this `<nav>` is, because you can have a million of them on a single page.

Reply:
just "site navigation"? 🤔

Reply:
Sure - it kind of sucks that PNI is just a bunch of pages instead of its own domain, making this way weirder than it has to be, but "site navigation" is good enough until we revisit.